pkg/collector/reader.go

```diff
@@ -397,3 +394,16 @@ func getActiveCPUs(cpuTime *[C.CPU_VECTOR_SIZE]uint16) (activeCPUs []int32) {
 	}
 	return
 }
+
+// handleInactivePods
+func handleInactivePods(foundPod map[string]bool) {
+	numOfInactive := len(podEnergy) - len(foundPod)
+	if numOfInactive > maxInactivePods {
```

Comment:
I am wondering if the could:
- have counter, if the pod did not appear for 3-5 consecutive epochs we need to verify if the pod still exist 
- when we verify if the pod exist, the `podLister.ListPods()` could have the pod name label, so that it will minimize the overhead

Reply:
I think we do not have to check specific missing pod but rather compute if the number garbage pods is more than the acceptable threshold. When it more than the acceptable threshold, we just check the alive pod from ListPod and remove the deleted ones.

Reply:
The problem with that is because we will report information of pods that are not running anymore

This means that if we sum the energy of all reported pods in the prometheus metrics, it will show a value higher than what RAPL is reporting, because we are still reporting the energy of pods that does not exist anymore....

Reply:
Hum, but since the current metric is always reset, maybe this might not be a problem.....
Wdyt?

Reply:
this could be a good stress test case to get some conclusions @jichenjc